ams/server/src/main/resources/postgres/ams-postgres-init.sql

```diff
@@ -0,0 +1,342 @@
+    optimizing_process_id BIGINT NOT NULL,
+    optimizer_group VARCHAR(64) NOT NULL,
+    table_config TEXT,
+    optimizing_config TEXT,
+    pending_input TEXT,
+    PRIMARY KEY (table_id),
+    UNIQUE (catalog_name, db_name, table_name)
+);
+COMMENT ON TABLE table_runtime IS 'Optimize running information of each table';
+COMMENT ON COLUMN table_runtime.table_id IS 'Table ID';
+COMMENT ON COLUMN table_runtime.catalog_name IS 'Catalog name';
+COMMENT ON COLUMN table_runtime.db_name IS 'Database name';
+COMMENT ON COLUMN table_runtime.table_name IS 'Table name';
+COMMENT ON COLUMN table_runtime.current_snapshot_id IS 'Base table current snapshot ID';
+COMMENT ON COLUMN table_runtime.current_change_snapshotId IS 'Change table current snapshot ID';
+COMMENT ON COLUMN table_runtime.last_optimized_snapshotId IS 'Last optimized snapshot ID';
+COMMENT ON COLUMN table_runtime.last_optimized_change_snapshotId IS 'Last optimized change snapshot ID';
+COMMENT ON COLUMN table_runtime.last_major_optimizing_time IS 'Latest Major Optimize time for all partitions';
+COMMENT ON COLUMN table_runtime.last_minor_optimizing_time IS 'Latest Minor Optimize time for all partitions';
+COMMENT ON COLUMN table_runtime.last_full_optimizing_time IS 'Latest Full Optimize time for all partitions';
+COMMENT ON COLUMN table_runtime.optimizing_status IS 'Table optimize status: FULL_OPTIMIZING, MAJOR_OPTIMIZING, MINOR_OPTIMIZING, COMMITTING, PENDING, IDLE';
+COMMENT ON COLUMN table_runtime.optimizing_status_start_time IS 'Table optimize status start time';
+COMMENT ON COLUMN table_runtime.optimizing_process_id IS 'Optimizing procedure UUID';
+COMMENT ON COLUMN table_runtime.optimizer_group IS 'Optimizer group';
+COMMENT ON COLUMN table_runtime.table_config IS 'Table-specific configuration';
+COMMENT ON COLUMN table_runtime.optimizing_config IS 'Optimizing configuration';
+COMMENT ON COLUMN table_runtime.pending_input IS 'Pending input data';
+
+CREATE TABLE table_optimizing_process
+(
+    process_id BIGINT NOT NULL,
+    table_id BIGINT NOT NULL,
+    catalog_name VARCHAR(64) NOT NULL,
+    db_name VARCHAR(128) NOT NULL,
+    table_name VARCHAR(128) NOT NULL,
+    target_snapshot_id BIGINT NOT NULL,
+    target_change_snapshot_id BIGINT NOT NULL,
+    status VARCHAR(10) NOT NULL,
+    optimizing_type VARCHAR(10) NOT NULL,
+    plan_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    end_time TIMESTAMP,
+    fail_reason VARCHAR(4096),
+    rewrite_input BYTEA,
+    summary TEXT,
+    from_sequence TEXT,
+    to_sequence TEXT,
+    PRIMARY KEY (process_id)
+);
+COMMENT ON TABLE table_optimizing_process IS 'History of optimizing after each commit';
+COMMENT ON COLUMN table_optimizing_process.process_id IS 'Optimizing procedure UUID';
+COMMENT ON COLUMN table_optimizing_process.table_id IS 'Table ID';
+COMMENT ON COLUMN table_optimizing_process.catalog_name IS 'Catalog name';
+COMMENT ON COLUMN table_optimizing_process.db_name IS 'Database name';
```

Comment:
There is an Index on (`table_id`, `plan_time`) in MySQL script.

Please help to check if this index is necessary.